components/service/experiments/README.md

```diff
@@ -73,12 +73,15 @@ In the above:
     |key|type|description|
     |---|----|-----------|
     | updateExperiments | boolean (--ez) | forces the experiments updater to run and fetch experiments immediately |
+    | setKintoInstance | string (--es) | sets the Kinto instance to the specified instance ("dev", "staging", "prod" only) |
 
-For example, to direct a release build of the Glean sample application to update experiments immediately, the following command
-can be used:
+For example, to direct a release build of the glean sample application to (1) update experiments immediately and 
```

Comment:
nit: we settled on `Glean` rather than `glean` :)

Reply:
oops ;)

Reply:
merged rolled right over it...